build-system-tests/templates/components/react/cra/App.js

```diff
@@ -46,6 +46,12 @@ export default function Home() {
         maxFileCount={1}
         isResumable
       />
+      <FileUploader
+        acceptedFileTypes={['image/*']}
+        accessLevel="guest"
```

Comment:
since this is the v5 version of the props, can we add v6 versions with path in this test as well for both FileUploader and StorageManager?

Reply:
```
    <StorageManager
      acceptedFileTypes={['image/*']}
      path="public/"
      maxFileCount={1}
      isResumable
    />
```

Reply:
https://ui.docs.amplify.aws/react/connected-components/storage/fileuploader#basic-usage

Reply:
updated